ts-boilerplate-common/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ts-boilerplate-common",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "tsc && node dist/index.js"
+  },
+  "author": "",
+  "license": "MIT",
+  "description": "",
+  "dependencies": {
+    "express": "^4.21.1"
```

Comment:
Raising a question here, what's the rationale for choosing Express as the default framework?

While Express is more widespread, it's a bit dated by today's standards and Fastify is arguably a better solution all around nowadays, especially when considering TypeScript support.
It's something we discussed in the past with Natalia, and the conclusion was that for new samples Fastify should be our default choice.